docs/how-to-guides/how-to-use-chainhook-with-bitcoin.md

```diff
@@ -122,8 +112,18 @@ Get any transaction including a p2sh output paying a given recipient `p2sh` cons
 }
 ```
 
-Get any transaction including a p2wpkh output paying a given recipient `p2wpkh` construct admits:
-- string type. example: "bcrt1qnxknq3wqtphv7sfwy07m7e4sr6ut9yt6ed99jg"
+```json
+{
+    "if_this": {
+        "scope": "outputs",
+        "p2sh": "0x76a914ee9369fb719c0ba43ddf4d94638a970b84775f4788ac"
+    }
```

Comment:
I'm blinking on this. Is this documentation correct @BLuEScioN?

Reply:
I will look into the following commits to the chainhook readme file and update the docs predicates.

https://github.com/hirosystems/chainhook/commit/8f726a16fe9fad2260feee85b0a4ef9f7521fac0
https://github.com/hirosystems/chainhook/commit/f9d7370c4398415fa4e1fcfc2de0b8c8c45d40f5
https://github.com/hirosystems/chainhook/commit/572cf202bae5adc37f5799789a2a68211d95486b